### PR TITLE
Densityclarify

### PR DESCRIPTION
**Pull request by sbird**
A few cosmetic cleanups in the Density code.

### CODE REVIEW

File: libgadget/density.c [hunk 1]
```diff
@@ -534,10 +532,10 @@ void density_check_neighbours (int i, TreeWalk * tw) {
 
 #ifdef BLACK_HOLES
         if(P[i].Type == 5)
-            if(DENSITY_GET_PRIV(tw)->Left[i] > All.BlackHoleMaxAccretionRadius)
+            if(P[i].Left > All.BlackHoleMaxAccretionRadius)
```
Comment (rainwoodman) [inline libgadget/density.c]: Left[i] and Right[i]. These are still the private members.

Comment (sbird) [reply libgadget/density.c]: Bah, that's what I get for ever removing BLACK_HOLES from my Options.mk

**Comment by sbird** [comment]
The CI is failing with this again:

CondaHTTPError: HTTP 504 GATEWAY_TIMEOUT for url <https://conda.anaconda.org/bccp/linux-64/halotools-0.6-py36h637b7d7_5.tar.bz2>

**Comment by rainwoodman** [comment]
the download error happens from time to time. Do you have the permission to trigger a rebuild?

**Comment by sbird** [comment]
Thanks!

**Comment by sbird** [comment]
Shall I merge it?

**Comment by rainwoodman** [comment]
Sure!

**Pull request merged**